javascriptv3/example_code/sts/README.md

```diff
@@ -1,56 +1,54 @@
-# AWS Security Token Service (AWS STS) JavaScript SDK v3 code examples
-AWS Security Token Service (AWS STS) as a web service that enables you to request temporary, 
-limited-privilege credentials for AWS Identity and Access Management (IAM) users or for users 
-you authenticate (federated users).
+# [AWS STS code examples for the SDK for JavaScript (v3)
 
-## Code examples
-This is a workspace where you can find the following AWS SDK for JavaScript version 3 (v3) Amazon IAM examples. 
+## Overview
 
-- [Assume an IAM role](src/sts_assumerole.js) (GetCallerIdentityCommand, AssumeRoleCommand)
+Shows how to use the AWS SDK for JavaScript (v3) with AWS Security Token Service (AWS STS).
 
-**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see
-[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-examples-javascript-syntax.html).
+AWS provides AWS STS as a web service that enables you to request temporary, limited-privilege credentials for AWS Identity and Access Management (IAM) users or for users you authenticate (federated users).
 
-## Important
+## ⚠️ Important
 
-- As an AWS best practice, grant this code least privilege, or only the
-  permissions required to perform a task. For more information, see
-  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the *AWS Identity and Access Management User Guide*.
-- This code has not been tested in all AWS Regions. Some AWS services are
-  available only in specific AWS Regions. For more information, see the
-  [AWS Regional Services List](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
-  on the AWS website.
 - Running this code might result in charges to your AWS account.
+- Running the tests might result in charges to your AWS account.
+- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+- This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Get started
```

Comment:
Is there a "Hello **service**" link for this category? If not, recommend deleting the "Get started" header for now.

Reply:
Not yet. This was a clean-up effort vs an MVP effort.